### ISSUE TITLE
Changing CPU governor cause OS crash

### ISSUE THREAD

**Issue by max20091** [latte-dev/android_kernel_xiaomi_latte#4]
As the title said, I tried to switch to powersave or performance governor cause the OS crash.

**Comment by tank0412** [comment]
Attach last_kmsg

**Comment by harrynowl** [comment]
Yes, please provide logs when creating issues. I can't reproduce this.

**Comment by max20091** [comment]
Oh here it is. it send a kernel panic message but I dont see anything suspicious
[last_kmsg.zip](https://github.com/latte-dev/android_kernel_xiaomi_latte/files/870023/last_kmsg.zip)

**Comment by max20091** [comment]
And why the CPU can turn on cores even I turned it off? (idk if this caused the kernel panic)

**Comment by tank0412** [comment]
It is OK for this CPU to disable all cores before entering a deep sleep.
It rebooted due to this:
[  345.254250] RIP  [<ffffffff818c1fc2>] cpufreq_bootboost_safety_timer+0x32/0x50
So maybe something wrong with this:
https://github.com/latte-dev/android_kernel_xiaomi_latte/blob/cm-13.0/drivers/cpufreq/cpufreq_interactive.c#L1468

Anyway, it is a really strange bug. 
I do not see any suspicious stuff in CPU Frequency scaling of defconfig.
